components/atom/checkbox/src/index.js

```diff
@@ -68,7 +72,10 @@ AtomCheckbox.propTypes = {
   onChange: PropTypes.func,
 
   /* Will set a red/green border if set to true/false */
-  errorState: PropTypes.bool
+  errorState: PropTypes.bool,
+
+  /* If you with you can change the look of checkbox using an AtomIcon */
```

Comment:
`If you with you` looks weird, i guess this is a unwanted mistake 🙄

Reply:
Done! 😅

Reply:
With or withooooou youuu